common/utils/src/main/resources/error/error-classes.json

```diff
@@ -2750,6 +2750,12 @@
     ],
     "sqlState" : "42000"
   },
+  "NULL_INDEX" : {
```

Comment:
The name confuses slightly. From my point of view, the error is about an invalid NULL value at the given position, how about:
```
NULL_ROW_VALUE
```